landa/water_body_management/doctype/stocking_target/stocking_target.js

```diff
@@ -3,8 +3,10 @@
 
 {% include "landa/water_body_management/stocking_controller.js" %}
 
+
 frappe.ui.form.on("Stocking Target", {
 	refresh: function (frm) {
+		landa.utils.set_default_year(frm);
```

Comment:
```suggestion
		if (frm.is_new() && !frm.doc.year) {
			landa.utils.set_default_year(frm);
		}
```

Sonst wird das Jahr immer überschrieben, auch wenn es manuell geändert wurde.